src/jackdaw/streams/configured.clj

```diff
@@ -52,6 +52,10 @@
     [_]
     (source-topics streams-builder))
 
+  (with-state-store
```

Comment:
I am not 100% on the naming here - maybe `add-state-store`. However this name make the code using it read nicer IMHO. `(-> builder (k/with-state-store ...) (k/stream ...))` etc.